tests/validator/validator.cpp

```diff
@@ -25,6 +25,15 @@ limitations under the License.
  * are not picked up by the main tests testing the API of the library
  */
 
+TEST(Validator, invalidInput)
+{
+    libcellml::ValidatorPtr validator = libcellml::Validator::create();
```

Comment:
Use `auto` rather than `libcellml::ValidatorPtr` (just to show "best practises").

Reply:
I thought we were to show alternative ways of doing things and not fixate on a single way of doing things in the tests, not in the library.

Reply:
I can see the benefit of doing that when it comes to libCellML, but can't see the point when it comes to C++. Anyway.